builder/azure/dtl/builder_acc_test.go

```diff
@@ -62,93 +65,8 @@ func TestDTLBuilderAcc_ManagedDisk_Linux_Artifacts(t *testing.T) {
 	})
 }
 
-const testBuilderAccManagedDiskWindows = `
```

Comment:
I had to change these templates to add the prefix variable, so I figured I'd just go ahead and extract these into template files